tests/pytests/test_replicate.py

```diff
@@ -211,3 +213,38 @@ def testDropWith__FORCEKEEPDOCS():
 
     env.assertEqual(master.execute_command('KEYS', '*'), ['doc1'])
     env.assertEqual(slave.execute_command('KEYS', '*'), ['doc1'])
+
+def testExpireDocs():
+  env = initEnv()
+  master = env.getConnection()
+  slave = env.getSlaveConnection()
+
+  '''
+  This test creates creates an index and two documents and check they
+  exist on both shards.
+  One of the documents is expired.
+  The test checks the document is removed from both master and slave.
```

Comment:
I do not see you use: `'DEBUG', 'SET-ACTIVE-EXPIRE', '0'`
The idea is to check active expire on slave as well